packages/nitro/src/builders.ts

```diff
@@ -68,3 +69,15 @@ export class LocalBuilder extends BaseBuilder {
     });
   }
 }
+
+export function getWorkflowDirs(nitro: Nitro) {
+  return unique([
+    ...(nitro.options.workflow?.dirs ?? []),
+    join(nitro.options.rootDir, 'workflows'),
+    ...nitro.options.scanDirs.map((dir) => join(dir, 'workflows')),
+  ].map((dir) => resolve(nitro.options.rootDir, dir))).sort();
```

Comment:
Is this resolve necessary here? I see these being resolved here as well: https://github.com/vercel/workflow/blob/94d46d423a1ef22593f4c11728cb5f60a916bb4b/packages/builders/src/base-builder.ts#L94-L100